app/registries/foreman/access_permissions.rb

```diff
@@ -305,10 +305,11 @@
                                     :puppetclasses => pc_ajax_actions,
                                     :subnets => subnets_ajax_actions,
                                     :interfaces => [:new, :random_name],
-                                    :registration => [:new, :create],
+                                    :registration_command => [:new, :create],
```

Comment:
From my experience it's not worth to change the name of the permission since it will require a migration to rename or delete the old one. Although I might be wrong, it's better to check in rails console.

Reply:
but this is controller name right? The permission is `:create_hosts`, see:
```ruby
    map.permission :create_hosts,  {:hosts => [:new, :create, :clone].push(*ajax_actions),
                                    :compute_resources_vms => [:import],
                                    :compute_resources => cr_ajax_actions,
                                    :puppetclasses => pc_ajax_actions,
                                    :subnets => subnets_ajax_actions,
                                    :interfaces => [:new, :random_name],
                                    :registration_command => [:new, :create],
                                     :"api/v2/hosts" => [:create],
                                     :"api/v2/interfaces" => [:create],
                                     :"api/v2/tasks" => [:index],
                                     :"api/v2/registration_command" => [:create],
                                     :"api/v2/registration" => [:global, :host]
```